### PR TITLE
Add libva build

### PR DESCRIPTION
**Pull request by adydychk**
In addition add some comments

### CODE REVIEW

File: conf_linux_public.py [hunk 1]
```diff
@@ -107,6 +109,22 @@ def get_building_cmd(command, gcc_latest, enable_devtoolset):
        cmd=print_gcc_version(GCC_LATEST, ENABLE_DEVTOOLSET),
        verbose=True)
 
+#Build dependencies
+##Build LibVA
+action('autogen.sh', #TODO: Do we have to build with latest gcc?
```
Comment (azhogov) [inline conf_linux_public.py]: compiler version should be the same as for the product

File: conf_linux_public.py [hunk 2]
```diff
@@ -141,6 +159,7 @@ def get_building_cmd(command, gcc_latest, enable_devtoolset):
        cmd=get_building_cmd(f'make DESTDIR={options["INSTALL_DIR"]} install', GCC_LATEST, ENABLE_DEVTOOLSET))
 
 
+#Form install and developer packages of MediaSDK
```
Comment (azhogov) [inline conf_linux_public.py]: libva should be packed as well

File: conf_linux_public.py (hunk 1 above)
Comment (adydychk) [reply conf_linux_public.py]: It is already implemented (see one line below)

File: conf_linux_public.py [hunk 3]
```diff
@@ -107,6 +109,28 @@ def get_building_cmd(command, gcc_latest, enable_devtoolset):
...
+       work_dir=LIBVA_REPO_DIR)
+
+action('list libva artifacts',
+       cmd=f'echo " " && ls ./va',
+       work_dir=LIBVA_REPO_DIR,
+       verbose=True)
+
+
+#TODO: build MediaSDK with fresh build LibVA (not system)(install / packaging?!)
+#export PKG_CONFIG_PATH=${PKG_CONFIG_PATH}:/usr/local/lib/pkgconfig
```
Comment (adydychk) [inline conf_linux_public.py]: @azhogov What do you think about connecting MediaSDK with NOT system libva?

File: conf_linux_public.py [hunk 4]
```diff
@@ -88,16 +88,18 @@ def get_building_cmd(command, gcc_latest, enable_devtoolset):
     else:
         return f'{enable_devtoolset} && {command}' #enable new compiler on CentOS
 
-
+#Repositories which will be downloaded
 PRODUCT_REPOS = [
     {'name': 'MediaSDK'},
+    {'name': 'libva'},
```
Comment (adydychk) [inline conf_linux_public.py]: clone libva with using the code which will be written here (because need to use manifest) or in opposite extend API

**Pull request closed without merging**